Assets/MixedRealityToolkit/Utilities/Gltf/Serialization/GltfConversions.cs

```diff
@@ -61,8 +77,16 @@ internal static float[] SetVector2Value(this Vector2 vector)
 
         internal static Vector3 GetVector3Value(this float[] vector3Array, bool convert = true)
         {
-            var vector = new Vector3(vector3Array[0], vector3Array[1], vector3Array[2]);
-            return convert ? Vector3.Scale(vector, CoordinateSpaceConversionScale) : vector;
+            if (vector3Array != null && vector3Array.Length == 2)
```

Comment:
Should this be 3, for a `Vector3`?

```suggestion
            if (vector3Array != null && vector3Array.Length == 3)
```